app/src/androidTest/java/ch/epfl/cs311/wanderwave/viewmodel/TrackListViewModelTest.kt

```diff
@@ -173,18 +173,18 @@ class TrackListViewModelTest {
     assertEquals("Failed to play track", viewModel.uiState.value.message)
   }
 
-  @Test
-  fun resumeTrackWhenControllerReturnsFalse() = run {
-    every { mockSpotifyController.pauseTrack() } returns flowOf(true)
-    every { mockSpotifyController.playTrack(track) } returns flowOf(true)
-    every { mockSpotifyController.resumeTrack() } returns flowOf(false)
-    viewModel.selectTrack(track)
-    viewModel.play()
-    viewModel.pause()
-    viewModel.play()
-    verify { mockSpotifyController.resumeTrack() }
-    assertEquals("Failed to resume track", viewModel.uiState.value.message)
-  }
+  //  @Test
+  //  fun resumeTrackWhenControllerReturnsFalse() = run {
+  //    every { mockSpotifyController.pauseTrack() } returns flowOf(true)
+  //    every { mockSpotifyController.playTrack(track) } returns flowOf(true)
+  //    every { mockSpotifyController.resumeTrack() } returns flowOf(false)
+  //    viewModel.selectTrack(track)
+  //    viewModel.play()
+  //    viewModel.pause()
+  //    viewModel.play()
+  //    verify { mockSpotifyController.resumeTrack() }
+  //    assertEquals("Failed to resume track", viewModel.uiState.value.message)
+  //  }
```

Comment:
You can safely delete this test, I wrote new ones on my branch that cover more than this one did anyway.